config.sample.js

```diff
@@ -138,6 +138,26 @@ module.exports = {
 			// RE: https://i.goopics.net/WRPWl.png
 			online: "XXXXXXXXXXX"
 			// RE: https://i.goopics.net/QdGW0.png
+		},
+		category: {
```

Comment:
I think it's better to call the object `categories` instead of `category` no?

Reply:
yes ;) !